client/components/data/query-site-comments-list/index.jsx

```diff
@@ -11,13 +12,15 @@ import { requestCommentsList } from 'state/comments/actions';
 
 export class QuerySiteCommentsList extends PureComponent {
 	static propTypes = {
+		context: PropTypes.string,
```

Comment:
this is an enum, we should probably consider making it obvious: `PropTypes.oneOf( [ 'display', 'edit' ] )`